browser/BrowserPerf.proto

```diff
@@ -25,36 +25,56 @@ import "common/common.proto";
 
 // Collect performance raw data from browser.
 service BrowserPerfService {
-    rpc collect (stream BrowserPerfData) returns (Commands) {
+
+    // report once per page
+    rpc collectPerfData (BrowserPerfData) returns (Commands) {
+    }
+
+    // report one or more error logs for pages, could report multiple times.
+    rpc collectErrorLogs (stream BrowserErrorLog) returns (Commands) {
     }
 }
 
 message BrowserPerfData {
+    int32 serviceId = 1;
+    // Service version in browser is the Instance concept in the backend.
+    int32 serviceVersionId = 2;
+    // Perf data time
+    int64 time = 3;
```

Comment:
If `time` field can be generated on server, could we remove `time` field?

Reply:
`time` and `pagePathId` need to be used in `browser-receiver`, please keep them.

Reply:
OK

Reply:
It is generated on the server only when `time` is empty.